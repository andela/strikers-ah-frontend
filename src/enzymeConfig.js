import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
