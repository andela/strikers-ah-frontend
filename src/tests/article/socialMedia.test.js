import React from 'react';
import { shallow } from 'enzyme';
import '../../enzymeConfig';

import {
  SocialMedia,
  mapStateToprops,
} from '../../components/article/SocialMedia';

const props = {
  article: {
    title: 'this is the title',
  },
};

const wrapper = shallow(<SocialMedia {...props} />);
describe('test for social media sharing tests', () => {
  test('should render the `SocialMedia` component', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('should test `mapstatetoprops`', () => {
    const mockedState = {
      Article: {
        article: {},
      },
    };
    const state = mapStateToprops(mockedState);
    expect(state).toBeTruthy();
  });
});
