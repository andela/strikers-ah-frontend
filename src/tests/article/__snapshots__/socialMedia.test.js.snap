// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for social media sharing tests should render the \`SocialMedia\` component 1`] = `ShallowWrapper {}`;
